.travis.yml

```diff
@@ -1,20 +1,22 @@
+dist: trusty
+sudo: false
 language: ruby
-cache:
-  directories:
-    - WcaOnRails/vendor/bundle
 rvm:
   - 2.3.1
+bundler_args: --path $HOME/.bundle
+cache:
+  directories:
+    - $HOME/.phantomjs
+    - $HOME/.bundle
 env:
   global:
-    - DATABASE_URL=mysql2://root@127.0.0.1:3306/wca_test
+    - DATABASE_URL=mysql2://root:travis@127.0.0.1:3306/wca_test
     - PATH=$HOME/.local/bin:$PATH
+    - RACK_ENV=test
 before_install:
-  # The Stripe API requires a newer version of openssl
-  - sudo apt-get update && sudo apt-get install --only-upgrade openssl
```

Comment:
Now that we're on ubuntu 14.04, we don't need to upgrade openssl anymore, which means we don't need sudo anymore! @pingskills check it out =)